cmd/update.go

```diff
@@ -0,0 +1,114 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+)
+
+var updateCmd = &cobra.Command{
+	Use: "update",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 2 {
+			// TODO: Improve this message
+			return fmt.Errorf("Invalid arguments, maximum 1 arguments possible")
+		}
+		return nil
+	},
+	Example: `# Change the source of a currently active component to local (use the current directory as a source)
```

Comment:
please add two spaces after '`' 

    Example: ` . # Change...

Reply:
@kadel Done

Reply:
spaces should be before `#` 😬

Reply:
check `ocdev update -h` output
```
Examples:
#  Change the source of a currently active component to local (use the current directory as a source)
  ocd update --local

  # Change the source of the frontend component to local with source in ./frontend directory
  ocdev update frontend --local ./frontend

  # Change the source of a currently active component to git
  ocdev update --git https://github.com/openshift/nodejs-ex.git

  # Change the source of the component named node-ex to git
  ocdev update node-ex --git https://github.com/openshift/nodejs-ex.git
```

Reply:
@kadel Done